src/main/java/ui/components/NavigableListView.java

```diff
@@ -45,7 +46,9 @@
     // (when nothing is selected, both should be no-ops)
     private boolean saveSelectionCalled = false;
 
-    private IntConsumer onItemSelected = i -> {};
+    public static final boolean IS_SECONDARY_KEY = true;
+
+    private BiConsumer<Integer, Boolean> onItemSelected = (index, rightKey) -> {};
```

Comment:
We should decide on the term we like better ('secondary key' or 'right mouse button'?) and be consistent everywhere.